src/ui/components/transactions/ScriptInvocation/ScriptInvocationCard.tsx

```diff
@@ -117,7 +96,19 @@ export class ScriptInvocationCard extends React.PureComponent<IProps> {
                   className={styles.expandableList}
                   showExpand={!collapsed}
                 >
-                  <AmountTable amounts={amounts} />
+                  <>
```

Comment:
You can skip wrapping in `<Fragment>` here

Reply:
`PlateCollapsable` component does not allow me to do this